jetstream/outcomes/firefox_desktop/networking.toml

```diff
@@ -67,3 +84,291 @@ log_space = true
+
+[metrics.cert_error_page_clicked.statistics.bootstrap_mean]
+
+[metrics.cert_error_page_clicked.statistics.deciles]
+
+[metrics.cert_error_page_clicked.statistics.kernel_density_estimate]
+
+[metrics.cert_error_page_clicked.statistics.empirical_cdf]
+
+
+[metrics.http_transactions_using_tls]
+select_expression = '{{agg_hist_zero_fract("payload.histograms.http_transaction_is_ssl")}}'
+data_source = "main"
+type = "scalar"
+
+[metrics.http_transactions_using_tls.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_transactions_using_tls.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_transactions_using_tls.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_transactions_using_tls.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.http_pageloads_using_tls]
+select_expression = '{{agg_hist_zero_fract("payload.histograms.http_pageload_is_ssl")}}'
+data_source = "main"
+type = "scalar"
+
+[metrics.http_pageloads_using_tls.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_pageloads_using_tls.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_pageloads_using_tls.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_pageloads_using_tls.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.http_channels_success]
+select_expression = '{{agg_hist_zero_fract("payload.histograms.http_channel_onstart_success")}}'
+data_source = "main"
+type = "scalar"
+
+[metrics.http_channels_success.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_channels_success.statistics.deciles]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_channels_success.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+
+[metrics.http_channels_success.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+
+
+[metrics.tls_successful_cert_validation_time]
+data_source = "main"
+select_expression = "ARRAY_AGG(mozfun.hist.extract(payload.processes.content.histograms.ssl_succesful_cert_validation_time_mozillapkix) IGNORE NULLS)"
+type = "histogram"
+bigger_is_better = false
+
+[metrics.tls_successful_cert_validation_time.statistics.bootstrap_mean]
+pre_treatments = ['remove_nulls']
+log_space = true
+
+[metrics.tls_successful_cert_validation_time.statistics.deciles]
+pre_treatments = ['remove_nulls']
+log_space = true
+
+[metrics.tls_successful_cert_validation_time.statistics.kernel_density_estimate]
+pre_treatments = ['remove_nulls']
+log_space = true
+
+[metrics.tls_successful_cert_validation_time.statistics.empirical_cdf]
+pre_treatments = ['remove_nulls']
+log_space = true
+
+
+[metrics.tls_failed_cert_validation_time]
+data_source = "main"
+select_expression = "ARRAY_AGG(mozfun.hist.extract(payload.processes.content.histograms.ssl_initial_failed_cert_validation_time_mozillapkix) IGNORE NULLS)"
+type = "histogram"
```

Comment:
Jetstream (which runs analyses on experiments) currently only support `scalar` metrics. `histogram` metrics are only supported in OpMon (which is used for monitoring rollouts or populations)